python/pypto/language/op/unified_ops.py

```
@@ -480,6 +483,32 @@ def row_min(input: T, tmp_tile: Tile | None = None) -> T:
     raise TypeError(f"row_min: expected Tensor or Tile, got {type(input).__name__}")
 
 
+def col_sum(input: T, tmp_tile: Tile | None = None) -> T:
```

Ruff (0.15.10)

[error] Function argument `input` is shadowing a Python builtin

(A002)